docs/api/session.md

```diff
@@ -259,6 +278,10 @@ The `proxyBypassRules` is a comma separated list of rules described below:
 
 Returns `Promise<String>` - Resolves with the proxy information for `url`.
 
+#### `ses.forceReloadProxyConfig()`
+
+Returns `Promise<void>` - Resolves when the proxy configuration is reloaded.
```

Comment:
This needs a description of what the method does.

What configuration is reloaded? From where?